README.mdown

```diff
@@ -432,3 +432,14 @@ Uses [go-lint](https://github.com/golang/lint) to lint go code.
 *Options*
 
 * `min_confidence` Set the confidence level of a problem before it is reported.
+
+### Lua
+
+#### Luacheck
+
+Uses [luacheck](https://github.com/mpeterv/luacheck) to lint Lua code.
+
+*Options*
+
+* None currently supported
```

Comment:
Should we support the `--config` option? That seems like a compact way for people to customize how luacheck works.